### DISCUSSION TITLE
Welcome to Delivroo Discussions!

### DISCUSSION THREAD

**Discussion by OmarChefrad** [OmarChefrad/Delivroo#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Astronier** [comment]
Hi Omar , i really like react native and the implementation of expo was on point good luck in your journey

**Comment by OmarChefrad** [comment]
ican't find the error ERROR MESSAGE at ScrollView

**Reply by Astronier** [reply]
All you need to do is to import it from "react-native" and keep the code clean if not try to restart the server
